include/fast_intdiv.h

```diff
@@ -16,6 +16,7 @@
 
 #ifndef _INT_FASTDIV_KJGIUHFG
 #define _INT_FASTDIV_KJGIUHFG
+#include<quda_arch.h>
```

Comment:
needed for float2 and double2 etc.

Reply:
Since this is a file from an external library, it would be better not to edit this directly.  Why wasn't this addition needed before?

Reply:
I believe it got included within another file after quda_internal.h . I added this here because one of the tests which included this file through another file didn't build.

Reply:
In that case, we need to reshuffle the headers where it needs to be done.  Alternatively, I could just move this file to the include/target/cuda/externals dir, and you can make an appropriate substitute for HIP.

Reply:
OK. I will check this once I switch to a faster build

Reply:
Reverted this in commit (0aa8f813d)